aikau/src/main/resources/alfresco/services/actions/CopyMoveService.js

```diff
@@ -201,6 +234,42 @@ define(["dojo/_base/declare",
                }
             }));
          }
+      },
+
+       /**
+        * Handles the actual create link XHR call
+        *
+        * @instance
+        * @param {boolean} copy A boolean indicating if this is a copy action or not
+        * @param {object} payload The payload from confirmation of the action dialog
+        */
+      onCreateLinkLocationSelected: function alfresco_services_ActionService__onCreateLinkLocationSelected(payload) {
+          this.alfUnsubscribeSaveHandles([this._actionCopyHandle]);
```

Comment:
This is where we'd pass `this._actionHandles` mentioned in the earlier comment... the `alfUnsubscribeSaveHandles` accepts an array, so we'd want to remove the square brackets.